src/xdsl/printer.py

```diff
@@ -164,20 +164,19 @@ def _print_results(self, op: Operation) -> None:
         # No results
         if len(results) == 0:
             return
-
         # One result
-        if len(results) == 1:
+        elif len(results) == 1:
```

Comment:
Why do you introduce an elif here? I find the following patterns very readable:

```
if ...
   return
if ...
   return
   
other 
instructions
return
```

Reply:
Yes, I agree as well.
This usually removes quite a lot of identation.

Reply:
agreed